spec/openapi-1.0.0.yml

```diff
@@ -0,0 +1,107 @@
+openapi: 3.0.0
+info:
+  version: '1.0.0'
+  title: 'DDB Statistics'
+  description: |
+    Systems within the DDB digital infrastructure can collected events occouring within the system and expose them for consumption by an external system responsible for performing statistical analysis.
+
+    This describes a set of APIs which should be exposed by a system exposing events.
+
+    Such a system is free to include this description and update descriptions to provide relevance to the current context.
+    Paths, arguments and event property names should not be altered.
+  license:
+    name: 'GNU General Public License v3.0'
+    url: 'https://www.gnu.org/licenses/gpl-3.0.html'
+
+security:
+  - BearerAuth: []
+paths:
+  /statistics:
+    patch:
+      operationId: claimStatistics
+      tags:
+        - List
+      description: 'Claim events occured within the system to be used for statistical purposes.'
+      parameters:
+        - schema:
+            type: string
+            format: date-time
+          in: query
+          name: since
+          description: |
+            From which point in time events should be returned. Defined as a datetime in ISO-8601 format.
+            By specifying a value an external system also signals that all preceeding events already have been claimed and can be deleted by the system.
+            If a value is not provided then all event collected by the system should be returned.
```

Comment:
Maybe be explicit about whether the specified date-time is included or excluded from the list. (`>` vs `>=`).